frontend/coreproject/src/lib/components/pages/anime_info/Index.svelte

```diff
@@ -1 +1,24 @@
-<p>Hello</p>
+<script lang="ts">
+  import { page } from '$app/stores';
+
+	import { anime_list } from '$data/mock/anime_list.ts';
+	import AnimeMainDetails from './AnimeMainDetails.svelte';
+
+  let anime_id = Number($page.params.id);
+
+	let anime = anime_list?.find((anime: any) => anime.id === anime_id);
+</script>
+
+<div class="anime_info overflow-hidden rounded-tl-3xl">
```

Comment:
`anime_info`

Why are we adding classes like this?

Reply:
Oky i will remove those, I added just to remember :|

Reply:
> Oky i will remove those, I added just to remember :|

Instead of doing this.

You can follow my approach

replace 

```html
<div></div>
```

with

```html
<anime-info></anime-info>
```

Reply:
Oky! Thanks for that